src/components/Question/Question.tsx

```diff
@@ -1,9 +1,49 @@
 import React  from 'react';
+import './Question.css'
+import bkg_img from '../../images/questions/Big Shoes - Sitting On Floor.png'
+import location_img from '../../images/questions/Charco - Location Map.png'
 export const Question: React.FC = () => {
   return (
-    <div>
-      {"hi"}
-    </div>
+    <section className='question-section'>
+      
+      <div className="inner-container">
+
+        <div className='desc-container'>
+          <div className="description-box">
+            <h1 className="question-desc">Yearly Salary After Taxes</h1>
+            <h2 className='desc'>"Gross income is the total amount you earn (typically over the course of a year) before expenses. Net income is the profit your business earns after expenses .</h2>
+            <h2 className='desc'>"Gross income is the total amount you earn (typically over the course of a year) before expenses. Net income is the profit your business earns after expenses .</h2>
```

Comment:
Looks good, should be pretty easy to conditionally render different info for different questions here

Reply:
Right. We'll just replace the hard coded messages with variables representing the context.